docs/src/api.md

```diff
@@ -1,20 +1,13 @@
 # Interface Functions
 
-Default methods cast all symbols to `Symbol` before comparing.
-
 ```@docs
-independent_variables
-is_indep_sym
-states
-state_sym_to_index
-is_state_sym
-parameters
-param_sym_to_index
-is_param_sym
+isvariable
+variableindex
+isparameter
+parameterindex
+isindependent_variable
+isobserved
+observed
+istimedependent
```

Comment:
the naming/underscore placement here seems slightly inconsistent

Reply:
`isindependentvariable` feels unreadable to me. Maybe `isindepvar`?

Reply:
Fixed in https://github.com/SciML/SymbolicIndexingInterface.jl/pull/12/commits/9bea16466126a7c57c8305c11597aea8beb190b7